samples/nrf9160/at_client/README.rst

```diff
@@ -0,0 +1,142 @@
+.. _at_client_sample:
+
+nRF9160: AT Client
+##################
+
+The AT Client sample demonstrates the asynchronous serial communication taking place over UART to the nRF9160 modem.
+The sample enables you to use an external computer or MCU to send commands to the LTE-M/NB-IoT modem of your nRF9160 DK.
+
+Overview
+********
+
+The AT Client sample acts as a proxy for sending directives to the nRF9160 modem via AT commands.
+This facilitates the reading of responses or analyzing of events related to the nRF9160 modem.
+The commands can be initiated from a terminal or the `LTE Link Monitor`_, which is an application implemented as part of `nRF Connect for Desktop`_.
+
+For more information on the AT commands, see the `AT Commands Reference Guide`_.
+
+Also see :ref:`modem_info_readme` for information about how an LTE application can interact with the modem.
```

Comment:
I am not sure to understand. Why is the `modem_info` module mentioned here ?

Reply:
Reference to the module has been removed already: https://github.com/NordicPlayground/fw-nrfconnect-nrf/pull/1345/files#r339546445

Reply:
Removed the reference.